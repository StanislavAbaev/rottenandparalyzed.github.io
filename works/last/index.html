<!DOCTYPE html>
<html lang="ru">
<head>
	<meta charset="UTF-8">
	<meta content="width=device-width, initial-scale=1" name="viewport" />
	<title>Document</title>
	<link href="https://fonts.googleapis.com/css?family=PT+Serif:400,700|Roboto:400,500&display=swap&subset=cyrillic" rel="stylesheet">
	<link rel="stylesheet" href="css/normalize.css">
	<link rel="stylesheet" href="css/fonts.css">
	<link rel="stylesheet" href="css/style.css">
	<link rel="stylesheet" href="css/jquery.fancybox.min.css">
	<link rel="stylesheet" type="text/css" href="css/slick.css"/>
	<link rel="stylesheet" type="text/css" href="css/jquery.formstyler.css">
	<link rel="stylesheet" href="css/media.css">
</head>
<body>
	<header class="header">
		
		<div class="header__top">
			<div class="container">

				<div class="header__contacts">
					<a href="tel:380963092145"class="header__phone">+38 (096) 309 21 45</a>
					<a href="#"class="header__email">layout585@gmail.com </a>
					<a data-fancybox data-src="#modal" href="javascript:;" class="header__btn">Бесплатная консультация</a>
				</div>
			</div>
		</div>
		<div class="container">
			<div class="header__container-inner">
				<div class="header__content">
					<div class="header__logo">
						<a href="#">
							<img src="img/logo.png" alt="">
						</a>
					</div>
				</div>
				<div class="icon-navicon"></div>
				<nav class="menu">
					<ul>
						<li><a href="#span">Главная</a></li>
						<li><a href="#">О компании</a></li>
						<li><a href="#">Услуги</a></li>
						<li><a href="#">Контакты</a></li>
					</ul>
				</nav>
				
			</div>
		</div>
	</header>

	<div id="modal">
		<form action="">
			<input type="text" placeholder="Ваше имя">
			<input type="text" placeholder="Ваш телефон">
			<input type="submit" value="Send">
		</form>
	</div>
	<section class="slider">
		<div class="container">
			<div class="slider__inner">
				<div class="slider__item">
					<div class="slider__item-content">
						<div class="slider__title">
							КОМПЛЕКСНОЕ ЗАКРЫТИЕ ПРЕДПРИЯТИЯ
						</div>
						<div class="slider__text">
							Полное прекращение существования юридического лица с сохранением легальности всей предыдущей деятельности.
						</div>
						<a data-fancybox data-src="#modal" href="javascript:;" class="slider__btn default-btn">
							Оставить заявку
						</a>
					</div>
				</div>
				<div class="slider__item">
					<div class="slider__item-content">
						<div class="slider__title">
							КОМПЛЕКСНОЕ ЗАКРЫТИЕ ПРЕДПРИЯТИЯ
						</div>
						<div class="slider__text">
							Полное прекращение существования юридического лица с сохранением легальности всей предыдущей деятельности.
						</div>
						<a data-fancybox data-src="#modal" href="javascript:;"  class="slider__btn default-btn">
							Оставить заявку
						</a>
					</div>
				</div>
				<div class="slider__item">
					<div class="slider__item-content">
						<div class="slider__title">
							КОМПЛЕКСНОЕ ЗАКРЫТИЕ ПРЕДПРИЯТИЯ
						</div>
						<div class="slider__text">
							Полное прекращение существования юридического лица с сохранением легальности всей предыдущей деятельности.
						</div>
						<a data-fancybox data-src="#modal" href="javascript:;"  class="slider__btn default-btn">
							Оставить заявку
						</a>
					</div>
				</div>

			</div>

		</div>
	</section>

	<section class="services">
		<div class="container">
			<div class="services__top">
				<div class="services__title-box">
					<div class="services-title">
						Наши Услуги
					</div>
					<div class="about__text">
						Комплексный подход к вашему вопросу, своевременная правовую помощь, представление интересов во всех судебных инстанциях.
					</div>
				</div>
				<div class="about__btn">
					<a  href="#">
						Показать все услуги
					</a>
				</div>
			</div>
			<div class="services__items">
				<div class="services__item">
					<img src="img/about-1.png" alt="">
					<div class="services__item-title">
						Банкротство и ликвидация
					</div>
					<div class="services__item-text">
						Полное прекращение существования юридического лица с сохранением легальности всей предыдущей деятельности
					</div>
					<div class="services__item-btn">
						<a href="#" class="services__item-link">Подробнее</a>
						<a data-fancybox data-src="#modal" href="javascript:;"  class="default-btn">
							Заказать
						</a>
					</div>
				</div>
				<div class="services__item">
					<img src="img/about-2.png" alt="">
					<div class="services__item-title">
						Адвокатское сопровождение
					</div>
					<div class="services__item-text">
						Мы гарантируем полное профессиональное обслуживание клиентов в заявленных сферах деятельности и соблюдение 
						конфиденциальности.
					</div>
					<div class="services__item-btn">
						<a href="#" class="services__item-link">Подробнее</a>
						<a data-fancybox data-src="#modal" href="javascript:;"  class="default-btn">
							Заказать
						</a>
					</div>
				</div>
				<div class="services__item">
					<img src="img/about-3.png" alt="">
					<div class="services__item-title">
						Налоговый аудит
					</div>
					<div class="services__item-text">
						Налоговый аудит проводят перед проведением налоговых проверок; при оценке возможных налоговых рисков; при оптимизации налогообложения.
					</div>
					<div class="services__item-btn">
						<a href="#" class="services__item-link">Подробнее</a>
						<a data-fancybox data-src="#modal" href="javascript:;"  class="default-btn">
							Заказать
						</a>
					</div>
				</div>
			</div>
		</div>
	</section>
	
	<section class="about">
		<div class="about__wrapper">
			<div class="container">
				<div class="about__inner">
					<div class="about__header">
						О компании 
					</div>
					<div class="about__text">
						Компания специализируется на оказании услуг в сфере корпоративного права, налогового консалтинга, представительства в судах, ликвидации и банкротства предприятий. На сегодняшний день, коллектив компании объединяет 
						высокопрофессиональных экспертов имеющих 
						специализации в отдельных областях права.
					</div>
					<div class="about__btn">
						<a data-fancybox data-src="#modal" href="javascript:;"  class="default-btn">
							Узнать больше
						</a>
					</div>
				</div>
			</div>
		</div>
	</section>

	<section class="form">
		<div class="container">
			<!-- <div class="form__inner"> -->
				
				<div class="form__title-box">
					<div class="form-title">
						Получить консультацию
					</div>
					<div class="form__text">
						Комплексный подход к вашему вопросу, своевременная правовую помощь, представление интересов во всех судебных инстанциях.
					</div>
				</div>
				<div class="form__box">
					<div class="form__box-inner">
						<div class="form__box-left">
							<label>
								E-mail
								<input type="text">
							</label>
							<label>
								Контактный телефон
								<input type="text">
							</label>
							<label>
								ФИО
								<input type="text">
							</label>
							<label>
								Тема вопроса
								<select>
									<option value="11">1dasd</option>
									<option value="22">221321</option>
									<option value="33">3gcx</option>
									<option value="44">4cccc</option>
								</select> 
							</label>
						</div>
						<div class="form__box-right">
							<label>
								Сообщение
								<textarea name="" id="" cols="30" rows="10"></textarea>
							</label>
							<button class="default-btn" type="submit" >Получить консультацию</button>
						</div>
					</div>
				</div>

			</div>
			<!-- </div> -->
		</section>
		
		<section class="advantages">
			<div class="advantages__wrapper">
				<div class="container">
					<div class="advantages__inner  ">
						<div class="advantages__box clearfix">
							<div class="advantages__box-check">
								
							</div>
							<div class="advantages__box-header">
								Десятки добровольных ликвидаций ежемесячно
							</div>
							<div class="advantages__text">
								Мы умеем минимизировать все риски в процессе добровольной ликвидации юридических лиц.
							</div>	
						</div>
						<div class="advantages__box">
							<div class="advantages__box-check">
								
							</div>
							<div class="advantages__box-header">
								Отличное знание законов в сфере банкротства 
							</div>
							<div class="advantages__text">
								Позволяют нам эффективно отстаивать как интересы кредиторов, так и интересы собственников проблемной организации.
							</div>	
						</div>
						<div class="advantages__box">
							<div class="advantages__box-check">
								
							</div>
							<div class="advantages__box-header">
								Большой опыт сопровождения банкротства
							</div>
							<div class="advantages__text">
								Мы обладаем большим опытом сопровождения процедур банкротства предприятий.
							</div>	
						</div>
					</div>
				</div>
			</div>
		</section>

		<section class="news">
			<div class="container">
				<div class="news__top">
					<div class="news__title-box">
						<div class="news-title">
							Новости компании
						</div>
						<div class="news__text">
							Комплексный подход к вашему вопросу, своевременная правовую помощь, представление интересов во всех судебных инстанциях.
						</div>
					</div>
					<div class="news__btn">
						<a  href="#">
							Открыть все новости
						</a>
					</div>
				</div>
				<div class="news__inner">
					<div class="news__slider">
						<div class="news__slider-inner">
							<div class="news__slider-item">
								<div class="news__slider-title">
									Банкротство и ликвидация предприятия в Украине
								</div>
								<div class="news__slider-text">
									«На сегодняшний день, коллектив компании объединяет высокопрофессиональных экспертов имеющих специализации в отдельных областях права. На сегодняшний день, коллектив компании объединяет высокопрофессиональных экспертов имеющих специализации в отдельных областях права. На сегодняшний день, коллектив компании объединяет высокопрофессиональных экспертов имеющих специализации в отдельных областях права...»
								</div>
								<div class="news__slider-author">
									Андрей Ворошилов, генеральный д иректор компании Bankom
								</div>
							</div>
							<div class="news__slider-item">
								<div class="news__slider-title">
									Банкротство и ликвидация предприятия в Украине
								</div>
								<div class="news__slider-text">
									«На сегодняшний день, коллектив компании объединяет высокопрофессиональных экспертов имеющих специализации в отдельных областях права. На сегодняшний день, коллектив компании объединяет высокопрофессиональных экспертов имеющих специализации в отдельных областях права. На сегодняшний день, коллектив компании объединяет высокопрофессиональных экспертов имеющих специализации в отдельных областях права...»
								</div>
								<div class="news__slider-author">
									Андрей Ворошилов, генеральный д иректор компании Bankom
								</div>
							</div>
							<div class="news__slider-item">
								<div class="news__slider-title">
									Банкротство и ликвидация предприятия в Украине
								</div>
								<div class="news__slider-text">
									«На сегодняшний день, коллектив компании объединяет высокопрофессиональных экспертов имеющих специализации в отдельных областях права. На сегодняшний день, коллектив компании объединяет высокопрофессиональных экспертов имеющих специализации в отдельных областях права. На сегодняшний день, коллектив компании объединяет высокопрофессиональных экспертов имеющих специализации в отдельных областях права...»
								</div>
								<div class="news__slider-author">
									Андрей Ворошилов, генеральный д иректор компании Bankom
								</div>
							</div>
						</div>
					</div>
					<a href='#' class="news__blog">
						<div class="news__images">
							<img src="img/news-1.jpg" alt="">
							<div class="news__date">03.04</div>
						</div>
						<div class="news__blog-title">
							Новость компании
						</div>
						<div class="news__blog-text">
							На сегодняшний день, коллектив компании объединяет высокопрофессиональных экспертов имеющих специализации в отдельных областях права...»
						</div>
					</a>
					<a href='#' class="news__blog">
						<div class="news__images">
							<img src="img/news-2.jpg" alt="">
							<div class="news__date">03.04</div>
						</div>
						<div class="news__blog-title">
							Новость компании
						</div>
						<div class="news__blog-text">
							На сегодняшний день, коллектив компании объединяет высокопрофессиональных экспертов имеющих специализации в отдельных областях права...»
						</div>
					</a>
				</div>
			</div>
		</section>
		
		<section class="contacts">
			<div class="container">
				<div class="contacts__inner">
					<div class="contacts__left">
						<div class="contacts__left-header">
							Комплексное закрытие предприятия
						</div>
						<div class="contacts__left-text">
							Полное прекращение существования юридического лица с сохранением легальности всей предыдущей деятельности.
						</div>
						<div class="contacts__data">
							<a data-fancybox data-src="#modal" href="javascript:;" class="contacts__btn">Бесплатная консультация</a>
							<a href="tel:380963092145"class="contacts__phone">+38 (096) 309 21 45</a>
							<a href="#"class="contacts__email">layout585@gmail.com </a>
							<a href="#"class="contacts__map-icon">Киев, ул.Пушиной, 13</a>
						</div>
					</div>
					<div class="contacts__right">
						<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2540.124391996489!2d30.364922015731644!3d50.457408279476354!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40d4cc919501b4ef%3A0x71a476f68f4c8246!2z0YPQuy4g0KTQtdC-0LTQvtGA0Ysg0J_Rg9GI0LjQvdC-0LksIDEzLCDQmtC40LXQsiwg0KPQutGA0LDQuNC90LAsIDAyMDAw!5e0!3m2!1sru!2sru!4v1567186484534!5m2!1sru!2sru" frameborder="0" style="border:0;" allowfullscreen=""></iframe>
					</div>
				</div>
			</div>
		</section>
		<footer>
			<span id="span">© 2017 Template by Anastasia Shaposhnyk. Все права защищены.</span>
		</footer>
		


		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
		<script type="text/javascript" src="js/slick.min.js"></script>
		<script src="js/jquery.fancybox.min.js"></script>
		<script src="js/jquery.formstyler.min.js"></script>
		<script src="js/main.js"></script>
	</body>
	</html>