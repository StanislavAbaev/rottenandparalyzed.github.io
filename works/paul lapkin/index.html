<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Paul Lapkin Homepage</title>
	<link rel="stylesheet" type="text/css" href="css/main.css">
	<link href="https://fonts.googleapis.com/css?family=Montserrat:400,900&display=swap" rel="stylesheet">
</head>
<body>
	<header>
		<div class="container">
			<div class="nav">
				<img src="img/logo.png" alt="">
				paul@lapkin.com
			</div>
			<div class="offer">
				Paul Lapkin
			</div>
			<div class="header__title-tags">
				#UserExperience  #Music  #Creative Direction
			</div>
		</div>
	</header>
	<section class="info">
		<div class="container">
			<div class="info__inner">
				<div class="info__left">
					<img src="img/man.jpg" alt="">
				</div>
				<div class="info__right">
					<div class="info__right-title">
						Hi, i’m Paul Lapkin, Experience Crafter from Canada.
					</div>
					<div class="info__right-text">
						<p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquipex ea commodo consequat.</p>
						
						<p>Duis autem vel eum iriure dolorin hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore.</p>
					</div>
					<div class="info__right-button">
						<a href="#" class="info__right-button">Hire me!</a>
					</div>
				</div>
			</div>
		</div>
	</section>
	<section class="process">
		<div class="container">
			<div class="process__title">
				My Working Process
			</div>
			<div class="process__subtitle">

				Hi, i’m Paul Lapkin, Experience Crafter from Canada.
			</div>
			<div class="process__inner">
				<div class="process__box blue">
					<img src="img/monitor.png" alt="">
					<p> design</p>
				</div>
				<div class="process__box purple">
					<img src="img/wand.png" alt="">
					<p>development</p>
				</div>
				<div class="process__box dark-blue">
					<img src="img/params.png" alt="">
					<p>testing</p>
				</div>
				<div class="process__box pink">
					<img src="img/plane.png" alt="">
					<p>launch</p>
				</div>
			</div>
		</div>
	</section>
	<section class="works">
		<div class="container">
			<div class="works__title">
				My Latest Works
			</div>
			<div class="works__subtitle">
				Hi, i’m Paul Lapkin, Experience Crafter from Canada.
			</div>
			<div class="works__inner">
				<div class="works__box blue"></div>
				<div class="works__box purple"></div>
				<div class="works__box dark-blue"></div>
				<div class="works__box pink"></div>
				<div class="works__box-long purple"></div>
				<div class="works__box dark-blue"></div>
				<div class="works__box pink"></div>
			</div>
		</div>
	</section>
	<footer class="footer">
		<div class="container">

				<ul class="socials">
					<li><a href="#">
						<i class="fab fa-twitter"></i>
					</a></li>
					<li><a href="#">
						<i class="fab fa-linkedin-in"></i>
					</a>
				</li>
					<li><a href="#">
						<i class="fab fa-behance"></i>
					</a>
				</li>
					<li><a href="#">
						<i class="fab fa-dribbble"></i>
					</a>
				</li>
			</ul>
			</div>
		<div class="footer__text">
			Copyright: Ali Sayed, All Rights Reserved.
		</div>


</footer>
<script src="https://kit.fontawesome.com/664f8ae0c0.js"></script>
</body>
</html>