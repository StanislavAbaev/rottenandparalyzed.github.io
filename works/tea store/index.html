<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Tea Store</title>
	<link type="text/css" rel="stylesheet" href="style.css">
</head>
<body>
<div class="main">
	<div class="logo">
		<img class="logo" src="img-tea-cozy-logo.png">
	</div>
		<div class="ul">
			<ul>
				<li class="li1"><a href="#"> Mission</a></li>
				<li class="li1"><a href="#">Feautured Tea</a></li>
				<li class="li1"><a href="#">Locations</a></li>
			</ul>
		</div>
</div>
	<div class=container>
		<div class=cont>
			<div class="image">
				<div class="text">
					<h2 class="text">Our Mission</h2>
					<h4 class="text">Handpicked, Artisanally Curated, Free Range, Sustainable, Small Batch, Fair Trade, Organic Tea</h4>
				</div>
			</div>
		</div>
	</div>
	<div class="mar">
		<h2> Tea of The Month</h2>
		<h3>What's Steeping at The Tea Cozy?</h3>
<div class=imgs>
		<div class="img">
			<img  src="img-berryblitz.jpg">
			<h4>Fall Berry Blitz Tea</h4>
		</div>
		<div class="img">
			<img  src="img-spiced-rum.jpg">
			<h4>Spiced Rum Tea</h4>
		</div>
		<div class="img">
			<img  src="img-donut.jpg">
			<h4>Seasonal Donuts</h4>
		</div>
		<div class="img">
			<img  src="img-myrtle-ave.jpg">
			<h4>Myrtle Ave Tea</h4>
		</div>
		<div class="img">
			<img src="img-bedford-bizarre.jpg">
			<h4>Bedford Bizarre Tea</h4>
		</div>
	</div>
</div>
 <div class="bottom">
	<div class="bottom-heading">
 		<!--<h3>Locations</h3>-->
	 </div> 
		<div class="bottom-box">
			<h4>Downtown</h4>
			<p>384 West 4th St</p>
			<p>Suite 308</p>
			<p>Portland, Maine</p>
		</div>
		<div class="bottom-box">		
			<h4>East Bayside</h4>     
			<p>3433 Phisherman's Avenue</p>
			<p>(Northwest Corner)</p>
			<p>Portland, Maine</p></div>
		<div class="bottom-box right">
			<h4>Oakdale</h4>
			<p>515 Crescent Avenue</p>
			<p>Second Floor</p>
			<p>Portland, Maine</p>
		</div>
</div>
<div class="end">
	<div class="end1">
		<div class="end-text">
			<h3 class="x">The Tea Cozy</h3>
			<h5 class="x">contact @theteacozy.com</h5>
			<a href="tel:9175558904" class="tel">917-555-8904</a>
			</div>
	</div>
</div>
<div class="foot">
<p><span>&copy</span>The Tea Cozy</p>
</div>
</body>
</html>