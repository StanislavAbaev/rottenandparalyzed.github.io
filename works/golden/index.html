<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet"  href="reset.css">
	<link rel="stylesheet" type="text/css" href="style.css">
</head>
<body>
	<header>
		<div class="container">
			<div class="ffr">
				<img src="img/logo.png" alt="" class="logo">
				<nav>
					<ul class="menu">
						<li><a href="#">HOME</a></li>
						<li><a href="#">SOURCES</a></li>
						<li><a href="#">PORTFOLIO</a></li>
						<li><a href="#">ABOUT</a></li>
						<li><a href="#">contact</a></li>		
					</ul>
				</nav>
			</div>
			<div class="titles">
				<h3 class="head-1">Welcome to our studio!</h3>
				<h1 class="head-2">It's nice to meet you!</h1>
			</div>
			<a id="button" href="#">Tell me more</a>
		</div>
	</header>
	<section>
		<div class="container">
			<div class="services">
				<h1 class="main-header">SERVICES</h1>
				<p class="under-header">Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="flex">
				<div class="box">
					<img src="img/icon1.png" class="img" alt="">
					<h4 class="small_header">E-Commerce</h4>
					<p class="small_text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>		
				</div>
				<div class="box">
					<img src="img/icon2.png" class="img" alt="">
					<h4 class="small_header">Responsive Web</h4>	
					<p class="small_text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="box">
					<img src="img/icon3.png" class="img" alt="">
					<h4 class="small_header">Web Security</h4>
					<p class="small_text">Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>	
				</div>
			</div>
		</div>
	</section>
	<section>
		<div class="container">
			<div class="services">
				<h1 class="main-header">OUR PORTFOLIO</h1>
				<p class="under-header">Proin iaculis purus consequat sem cure.</p>
			</div>
			<div class="flex">
				<div class="box">
					<img src="img/img1.jpg" class="img" alt="">	
				</div>
				<div class="box">
					<img src="img/img2.jpg" class="img" alt="">
				</div>
				<div class="box">
					<img src="img/img3.jpg" class="img" alt="">
				</div>
			</div>
		</section>
		<section>
			<div class="container">
				<div class="about">
					<h1 class="main-header">ABOUT US</h1>
					<p class="under-header">Proin iaculis purus consequat sem cure.</p>
				</div>
				<div class="about__inner">
					<div class="about__box">
						<img src="img/about1.jpg" alt="">
						<p class="text-leftside text">
							<span class="mounth">July 2010</span><br>
							<span class="subheader">Our Humble Beginings</span><br>
						Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos tempore similique fugit alias, tenetur quod id dolorum veniam doloremque quisquam repellat temporibus, excepturi at placeat? Dolore assumenda fugiat sequi at.</p>
					</div>

					<div class="about__box">
						<img src="img/about2.jpg" alt="" class="img1">
						<p class="text-rightside text">
							<span class="mounth">January 2011</span><br>
							<span class="subheader">Facing Startup Battles</span><br>
							Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos tempore similique fugit alias, tenetur quod id dolorum veniam doloremque quisquam repellat temporibus, excepturi at placeat? Dolore assumenda fugiat sequi at.
						</p>
					</div>
					<div class="about__box">
						<img src="img/about3.jpg" alt="">
						<p class="text-leftside text">
							<span class="mounth">December 2012</span><br>
							<span class="subheader">Enter The Dark Days</span><br>
						Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos tempore similique fugit alias, tenetur quod id dolorum veniam doloremque quisquam repellat temporibus, excepturi at placeat? Dolore assumenda fugiat sequi at.</p>
					</div>
					<div class="about__box">
						<img src="img/about4.jpg" alt="">
						<p class="text-rightside text">
							<span class="mounth">February 2014</span><br>
							<span class="subheader">Our Triumph</span><br> 
						Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos tempore similique fugit alias, tenetur quod id dolorum veniam doloremque quisquam repellat temporibus, excepturi at placeat? Dolore assumenda fugiat sequi at.</p>
					</div>
				</div>
			</div>
		</div>
	</div>
</section>
</body>
</html>